appveyor.yml

```diff
@@ -64,6 +65,8 @@ build_script:
   - "copy hooks\\pyi_rth*.* %PYTHON%\\Lib\\site-packages\\PyInstaller\\loader\\rthooks\\."
   # Replace the stage 3 importer
   - "copy hooks\\pyimod03_importers.py %PYTHON%\\Lib\\site-packages\\PyInstaller\\loader\\."
+  # Replace the part of the build script to avoid storing project path names
+  - "copy hooks\\utils.py %PYTHON%\\Lib\\site-packages\\PyInstaller\\building\\."
```

Comment:
Am I right that you're dropping in a modified version of an existing utils.py?  Why not patch it instead?